apps/mobile-wallet/src/components/Row.tsx

```diff
@@ -77,7 +82,12 @@ const Row = ({
   )
 
   return onPress ? (
-    <AnimatedPressable onPress={onPress} style={style}>
+    <AnimatedPressable
+      onPress={onPress}
+      onTouchStart={handleTouchStart}
+      onTouchEnd={handleTouchEnd}
+      style={[style, { opacity: isPressed ? 0.8 : 1 }]}
```

Comment:
just curious, shouldn't this be a shared value to avoid re-rendering? Maybe I'm wrong

Reply:
I've used a shareValue in doubt